app/controllers/flows_controller.rb

```diff
@@ -596,6 +596,7 @@ def self.common_attributes
     def self.ny_attributes(first_name: 'Testuser', last_name: 'Testuser')
       common_attributes.merge(
         confirmed_permanent_address: "no",
+        confirm_third_party_designee: "unfilled",
```

Comment:
Minor suggestion to rename to `confirmed_third_party_designee` to be consistent with `confirmed_permanent_address` - it is past tense because once it has been set to yes that means the confirmation has already happened.